modules/BuildUtilities/src/server/routes/RequiredClientScripts_template.js

```diff
@@ -1,21 +1,22 @@
 exports.requiredScripts = [
 	'/lib/jquery.min.js',
+	'/lib/jquery-migrate-1.4.1.js', // Remove after upgrading
```

Comment:
We can remove this, right?

Did you end up installing jquery-migrate 3.x and working through the warnings it showed also?

Reply:
I did install it, I fixed a few things but chose not to remove it as I figured we had some more testing to do.  I can remove it before merging if you like. What do you think?

Reply:
Let's remove it here. jquery-migrate 1.x doesn't work with jquery 3.x which this branch already gets us to. Makes sense to do some more testing after merging this, but we should use jquery-migrate 3.x anyway, and I think you and I can do that without checking in that library.